### DISCUSSION TITLE
log10

### DISCUSSION THREAD

**Discussion by WilliamDrap** [JetBrains/intellij-micropython#222] [Q&A] [answered]
hello,
don't know if it's the good place to post but ... 
try to use math.log10 function with pycharm and the micropython plugin. Pycharm accept the 'from math import log10' syntax but in REPL the message : can't import name 'log10' is displayed.
Any idea ?
Thank you for advance
William

**Comment by Elizaveta239** [comment]
Hi @WilliamDrap! Sorry for the late response! Which device and plugin version are you using?

**Reply by elmot** [reply, marked as answer]
That is obviously Micropython issue or unsupported issue. Please consult their docs
The plugin has nothing to do with it.
https://docs.micropython.org/en/latest/library/math.html
